api/src/routing.js

```diff
@@ -336,10 +337,22 @@ app.get('/api/info', function(req, res) {
   res.json({ version: p.version });
 });
 
-app.get('/api/deploy-info', (req, res) => {
+app.get('/api/deploy-info', async (req, res) => {
   if (!req.userCtx) {
     return serverUtils.notLoggedIn(req, res);
   }
+
+  // todo move this to some service or something
```

Comment:
If I add it to `environment`, it will create a circular dependency, where `db` requires `environment` and `environment` requires `db`. This is the only reason it isn't there and this silly todo comment. 
Do you think I should just require `db` at runtime, in the `environment` function?

Reply:
D'oh! No, I'm not a fan of runtime requires. Leave it as is or make a separate service, all good.

Reply:
I had already added a separate service :)